.github/workflows/push_dist_to_npm.yml

```diff
@@ -0,0 +1,31 @@
+name: Push dist to npm
+on: push
+
+jobs:
+  publish:
+    if: github.repository_owner == 'graphhopper'
+    runs-on: ubuntu-22.04
+    environment: npm
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          registry-url: https://registry.npmjs.org/
+          node-version: lts/gallium
+
+      - name: Build
+
+        run: |
+          npm version --git-tag-version=false 0.0.0-$(git rev-parse HEAD)
```

Comment:
npm requires a certain version string format. But since we want to use the git sha as version I set this to `0.0.0-<git sha>`.